deploy/charts/operator/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: toolhive-operator
 description: A Helm chart for deploying the ToolHive Operator into Kubernetes.
 type: application
-version: 0.3.1
+version: 0.3.2
```

Comment:
How come we are bumping the helm chart here? Nothing has changed in this chart for us to release?